internal/database/database.go

```diff
@@ -0,0 +1,47 @@
+package database
+
+import (
+	"context"
+	"database/sql"
+	"log"
+
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	_ "github.com/jackc/pgx/v5/stdlib"
+)
+
+type PostgresDatabase struct {
+	DB *sql.DB
+}
+
+func NewDB(ctx context.Context, ps string) (*PostgresDatabase, error) {
```

Comment:
```suggestion
func NewDB(_ context.Context, ps string) (*PostgresDatabase, error) {
```
а в принципе он тебе тут точно нужен будет?

Reply:
не нужен, убрала

Reply:
да, не нужен. Убрала